### DISCUSSION TITLE
Description of @reconnect in simulation manual

### DISCUSSION THREAD

**Discussion by haug-den-lucas** [omnetpp/omnetpp#1204] [Ideas]
Hello,

I just randomly found out about the `@reconnect` feature of OMNeT++ and realized there is no description of it in the Simulation Manual.
As this is quite an helpful feature, especially when extending modules from another project I think it might be helpful to add a short description and example of it to the Simulation Manual.

**Comment by rhornig** [comment]
thanks. converted to issue #1210
